### PR TITLE
#14083 Add VatCode/Tax Rate Name to Accounts Information Reports

### PR DESCRIPTION
**Pull request by Chemssou**
https://github.com/metasfresh/metasfresh/issues/14083

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
-initial_balance=Anfangssaldo
-sum=Summe
-sum_eur=Summe in EUR
\ No newline at end of file
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+account = Konto
+
+accountno = Konto-Nr.
+
```
Comment (cristinamghita) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties:6]: What's this?
Why did you modify it?

**Review by cristinamghita — CHANGES_REQUESTED** [review]
Seems fine, except for the properties files.

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties (hunk 1 above)
Comment (Chemssou) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties:6]: @cristinamghita  
this was generated/reordrerd by : `#Generated by Eclipse Messages Editor (Eclipse Babel)`

**Review by cristinamghita — CHANGES_REQUESTED** [review]

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties (hunk 1 above)
Comment (Chemssou) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/account_info/report_de_DE.properties]: fixed

**Pull request merged**